most_active_mentors.py

```diff
@@ -0,0 +1,149 @@
+"""A module for measuring the number of very active mentors
+
+This module provides functions for measuring the number of active mentors on a project.
+
+This is measured by number of PR comments. We are working under the assumption that PR
+comments are left in good faith to move contributors further instead of nitpicking and
+dis-couraging them.
+
+Open questions:
+   - should there be an option to limit this to certain users, e.g. core maintainers?
+   - should there be a limit to how many comments per PR we consider to avoid having
+     the statistic dominated by contested PRs?
+   - should this metric count consecutive comments coming from the same user as only
```

Comment:
Great question. In this case I would also consider the length of the comment. If somebody has a lot to say and they break that up into several meaningful comments, I would see that as several contributions worth counting.